### PR TITLE
Ingest data at startup

### LINKED ISSUE #43: Sample dashboard and example druid data to be ingested at dev env start
Currently, included dashboard that is based on example druid data does not work when one does `./mage env:start`. It is a very common question that noobs ask [(see discussion here)](https://grafadruid.slack.com/archives/C01F0503JQZ/p1613574299026500). It will be nice if we can import the dashboard at startup and also ingest the example data when druid container comes up.

### PR DESCRIPTION
**Pull request by raakasf**
close #43 

- Downloads druid scripts to ingest data
- Automatically ingests example data
- Provisions example datasource and dashboard

### CODE REVIEW

File: docker/grafana/provisioning/dashboards/druid.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: 1
+
+providers:
+  - name: 'a unique provider name'
```
Comment (jbguerraz) [inline docker/grafana/provisioning/dashboards/druid.yml:4]: grafadruid maybe ? :)

File: docker/toolbox/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM node:14.15.4
+ENV PATH="${PATH}:/usr/local/go/bin:/home/node/go/bin"
+ENV CGO_ENABLED=0
+RUN apt-get install git curl && curl -O https://dl.google.com/go/go1.14.4.linux-amd64.tar.gz && tar -xvf go1.14.4.linux-amd64.tar.gz && chown -R root:root ./go && mv go /usr/local
+
+# This section help ingest example data from Druid website to local instance
```
Comment (jbguerraz) [inline docker/toolbox/Dockerfile:6]: why not running those commands from the magefile target ?
mage env:provision or so ?

File: docker/grafana/provisioning/dashboards/druid.yml (hunk 1 above)
Comment (raakasf) [reply docker/grafana/provisioning/dashboards/druid.yml:4]: Oh yeah, missed out that one.

**Comment by raakasf** [comment]
@jbguerraz I am done with all the changes.

**Review by jbguerraz — APPROVED** [review]
looks cool :) :+1: :100: 
Thank you @raakasf !

**Pull request merged**